src/themes/dark/styles.ts

```diff
@@ -0,0 +1,49 @@
+import { colors } from './colors';

export const styles = {
```

Comment:
same line, if possible change before merge

Reply:
For some reason the line breaks my code generates don't show up here 😭 

If you look at the raw file you can see the breaks but not here for some reason. Tried a few things but no change so I think we may have to leave it for now.

Reply:
Wait... no, I screwed up. The new line breaks I try work... I just didn't compile my sketch plugin after I updated it and so I was just recompiling the same thing all the time.